res/layout/list_fragment.xml

```diff
@@ -40,19 +48,14 @@
         android:layout_height="match_parent"
         android:visibility="gone" >
 
-        <ScrollView
```

Comment:
I think this message should continue appearing when the folder is empty, so, maybe, we should keep it.